model/mustache.go

```diff
@@ -78,6 +78,14 @@ func (g *InstanceGroup) GetVariablesForRole() (Variables, error) {
 		}
 	}
 
+	configs["KUBERNETES_CONTAINER_NAME"] = &VariableDefinition{
+		Name: "KUBERNETES_CONTAINER_NAME",
+		CVOptions: CVOptions{
+			Type:    CVTypeEnv,
+			Default: g.Name,
```

Comment:
So the container name is the name of the instance group, not the container? Oh, because the "instance group" here does _not_ include the addons / colocated containers.

We should probably include a comment here so I don't get confused again in the future…

Reply:
The co-located container has its own instance group; it is not part of the main instance group:

https://github.com/SUSE/scf/blob/develop/container-host-files/etc/scf/config/role-manifest.yml#L2590-L2591

What do you want the comment to say?

Reply:
Something to indicate that the `instance group` here _isn't_ the pod name because…